### DISCUSSION TITLE
does iina supports them?

### DISCUSSION THREAD

**Discussion by neakk** [iina/iina#4873] [Q&A]
Does iina support hdr10, dolby vision, dts audio and dolby digital with doby atmos audio codec over 2 channel?

plz tell me about supported codec on iina.

and thx for the effort to make amazing app

**Comment by low-batt** [comment]
For audio/video playback support IINA is using a library from the [mpv](https://mpv.io/) project. As a result most of these questions are effectively about the capabilities of `mpv`. You may be able to obtain better answers by searching the net for "mpv player" and searching open `mpv` [issues](https://github.com/mpv-player/mpv/issues).

The IINA specific portion of this has to do with HDR. HDR10 should be working.

There are issues with Dolby Vision. I think proper support for Dolby Vision requires [gpu-next](https://github.com/mpv-player/mpv/wiki/GPU-Next-vs-GPU#new-functionality). That is the next generation renderer. Before IINA can use that the `mpv` project needs to add support for it to their render API.

Audio is fully handled by `mpv`. I'm not the best person to answer audio questions.
